### ISSUE TITLE
Interpolation indices go into negatives and crash

### ISSUE THREAD

**Issue by jpcima** [sfztools/sfizz#751]
`01-Standard Guitar KSOP.sfz` and MIDI fuzz LV2

Sometimes the indices and coeffs get invalid values and produce a crash.
It might have to do with recent updates of the loop code.

Below a sequence of (index, coeffs) values preceding a crash
```
5000 0.000000
5001 0.000000
5002 0.000000
5003 0.000000
5004 0.000000
5005 0.000000
5006 0.000000
5007 0.000000
5008 0.000000
5009 0.000000
5010 0.000000
5011 0.000000
5012 0.000000
5013 0.000000
5014 0.000000
5015 0.000000
5016 0.000000
5017 0.000000
5018 0.000000
5019 0.000000
5020 0.000000
5021 0.000000
5022 0.000000
5023 0.000000
5024 0.000000
5025 0.000000
5026 0.000000
5027 0.000000
5028 0.000000
5029 0.000000
5030 0.000000
5031 0.000000
-2147478648 12917102592.000000
-2147478648 23633942528.000000
-2147478648 34298261504.000000
-2147478648 44910346240.000000
-2147478648 55470448640.000000
-2147478648 65978773504.000000
-2147478648 76435628032.000000
-2147478648 86841262080.000000
-2147478648 97195900928.000000
-2147478648 107499814912.000000
-2147478648 117753241600.000000
-2147478648 127956434944.000000
-2147478648 138109599744.000000
-2147478648 148213039104.000000
-2147478648 158266949632.000000
-2147478648 168271642624.000000
-2147478648 178227314688.000000
-2147478648 188134195200.000000
-2147478648 197992546304.000000
```

**Comment by jpcima** [comment]
It seems the pitch modulation gets in very high values (like 38639 cents)

**Comment by jpcima** [comment]
Fixed by #825
